pkg/kubelet/stats/helper.go

```diff
@@ -433,3 +435,29 @@ func addUsage(first, second *uint64) *uint64 {
 	total := *first + *second
 	return &total
 }
+
+func makePodStorageStats(s *statsapi.PodStats, rootFsInfo *cadvisorapiv2.FsInfo, resourceAnalyzer stats.ResourceAnalyzer, hostStatsProvider HostStatsProvider, isCRIStatsProvider bool) {
+	podNs := s.PodRef.Namespace
+	podName := s.PodRef.Name
+	podUID := types.UID(s.PodRef.UID)
+	vstats, found := resourceAnalyzer.GetPodVolumeStats(podUID)
+	if !found {
+		return
```

Comment:
I don't think this function should require Volume stats to be present... the old code didn't require them either.

Reply:
why not? CRI stats code did, and seems to be working fine https://github.com/kubernetes/kubernetes/pull/108855/files#diff-27608c014abc817b90c50f8bbca712d16caa731c5d12465435cde848c72c2dc9L483